desktop/apps/auction2/components/artwork_browser/main/artwork/ListArtwork.js

```diff
@@ -20,8 +19,8 @@ function ListArtwork (props) {
     title
   } = props
 
-  const b = block('auction2-page-list-artwork')
-  const auctionArtworkClasses = classNames(b(), { 'auction2-open': isClosed })
+  const b = block('auction2-page-ListArtwork')
+  const auctionArtworkClasses = classNames(String(b()), { 'auction2-open': isClosed })
```

Comment:
React's `className` can take a string or a function but unfortunately `classNames` only takes a string. `bem-cn` is designed with a fluent interface for chaining, returning a function each time.